src/EFX_common/EffectLFO.C

```diff
@@ -137,6 +155,37 @@ EffectLFO::updateparams(uint32_t period)
     tca = iperiod / (iperiod + tmp); //
     tcb = 1.0f - tca;
     maxrate = 4.0f * iperiod;
+    
+    float slider_range = ui_freq_max - ui_freq_min; // avoid div by zero
+    if (slider_range < 1)  slider_range = 1;
+    float sep = CLAMP (((float) (Pstereo + 64.0f) / 128.0f) - 1.0f, -0.5, 0.5);
```

Comment:
stupid tab/indentation glitch, took 2 git commits in attempting to fix it. :( sorryyy...